spec/cypress/end_to_end/research_plan_with_user.cy.js

```diff
@@ -13,4 +13,36 @@ describe('Research Plan with User', () => {
     cy.contains('My Research Plan 2');
     cy.get('#tabList-tab-4 > span').contains('1(0)');
   });
+
+  it('rename research plan', () => {
```

Comment:
This test doesn't contain any assertion? Or is the `.click()` considered an (implicit) assertion?

Reply:
My intention was here to check the click functionality, because it was a bug there, the component created but not removed from the list. But yeah, adding assertion here would also be a good idea.